hid_enabled.go

```diff
@@ -227,3 +274,46 @@ func (dev *Device) Read(b []byte) (int, error) {
 	}
 	return read, nil
 }
+
+// GetFeatureReport retreives a feature report from a HID device
+//
+// Set the first byte of []b to the Report ID of the report to be read. Make
+// sure to allow space for this extra byte in []b. Upon return, the first byte
+// will still contain the Report ID, and the report data will start in b[1].
+func (dev *Device) GetFeatureReport(b []byte) (int, error) {
+	// Abort if nothing to read
```

Comment:
Please make this comment more accurate: there might be something to read, it's simply that there is no provided space to copy it to.

Reply:
Done.